assets/dev/js/frontend/frontend.js

```diff
@@ -14,6 +14,7 @@ import Breakpoints from 'elementor-utils/breakpoints';
 import Events from 'elementor-utils/events';
 import Shapes from 'elementor/modules/shapes/assets/js/frontend/frontend';
 import Controls from './utils/controls';
+import ConversionCenter from 'elementor/modules/conversion-center/assets/js/frontend';
```

Comment:
Load this behind the Experiment in this file:

https://github.com/elementor/elementor/blob/main/assets/dev/js/frontend/elements-handlers-manager.js#L33-L35

Reply:
this is done. Thanks for your help!!